docs/game/cheat_sheet.md

```diff
@@ -60,13 +65,14 @@ Deviation:
 Tactics
 -------
 
-* Speaking in a slow, measured way is beneficial. Just don't slow down to a stop.
+* Speaking in a measured way is beneficial, although be careful not to slow down too much - try and maintain your normal speaking voice and variation in tone.
```

Comment:
We may need to rethink this line. Some of the most entertaining runs in the 'real world' are when someone dons a character, or intentionally changes their voice in some way. (Case in point, Paul Merton recently did a whole round in an impression of Ringo Starr). 

While I realise that I'm not working with professional entertainers, I don't want to discourage that kind of behaviour.